client/tsconfig.json

```diff
@@ -18,6 +18,7 @@
 	"references": [ { "path": "../packages" }, { "path": "../build-tools" } ],
 	"include": [
 		"**/*",
+		"src/**/*",
```

Comment:
Without adding this, imports within `client` that did something like:

```
import {xyz} from 'calypso/...';
```

would throw a warning about an import (which I am having a hard time locating what the issue was about).

Reply:
I'd like to hear more about this TBH.

Reply:
I initially included this for the reason cited above - it was to fix a whole bunch of import warnings for imports within `client`, but I honestly am a bit skeptical that I should do this.

I'll first remove this particular change from the PR and see if the import warnings resurface.